packages/cdk/apps/core/app.ts

```diff
@@ -28,17 +53,7 @@ new CoreStack(app, `${PRODUCT_NAME}-Core`, {
   tags: {
     [APP_TAG_KEY]: APP_NAME,
     [AGC_VERSION_KEY]: agcVersion,
+    ...customTagsMap, // Add tags here so all infra in the core stack will be tagged as well.
```

Comment:
same thing, I think you want to spread the customTags at the top. 

But this also means we will override any custom tags with our own tags if they use the same key